src/reflect/scala/reflect/runtime/SynchronizedOps.scala

```diff
@@ -27,7 +27,6 @@ private[reflect] trait SynchronizedOps extends internal.SymbolTable
   trait SynchronizedBaseTypeSeq extends BaseTypeSeq {
     override def apply(i: Int): Type = gilSynchronized { super.apply(i) }
     override def rawElem(i: Int) = gilSynchronized { super.rawElem(i) }
-    override def typeSymbol(i: Int): Symbol = gilSynchronized { super.typeSymbol(i) }
```

Comment:
I can restore this, but remove the lock which hasn't been necessary since 2.12.0, due to other refactorings.